NOT_SUPPORTED_USE_CASES.md

```diff
@@ -0,0 +1,45 @@
+# Rare cases in which backups are not supported by BBQ
+
+In case of some rare circumstances BBQ is not able to verify backup last update time. Such situation results in not creating most up to date backup or creating wrong backup.
+
+## `lastModifiedTime` changed to past value when chunk of data is deleted 
+
+When part of BigQuery table/partition is deleted and deleted part was whole chunk of BigQuery internal storeage, then `lastModifiedTime` is max value taken from other chunks which might be earlier than `lastModifiedTime` of deleted chunk.
```

Comment:
also I believe that this problem is about partition only, as whole table metadata seems to bo stored physically for each table and updated in that case (partition metadata are calculated)

Reply:
As agreed we leave as it is